docs/guides/dev-server/storybook.md

```diff
@@ -26,7 +26,7 @@ export default {
 Add a `.storybook/main.js` file:
 
 ```js
-module.exports = {
+export default {
```

Comment:
it probably works because you added `ts-node/esm` which changed the behavior of NodeJS require internals, but I don't think it's a good idea to do this, because then you are not testing how the majority of people run dev-server-storybook/Storybook6